.github/workflows/build-image-ui-pr.yml

```diff
@@ -0,0 +1,25 @@
+name: Test UI container image build and deployment
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "clients/ui/**"
+      - "!LICENSE*"
+      - "!DOCKERFILE*"
+      - "!**.gitignore"
+      - "!**.md"
+      - "!**.txt"
+env:
+  IMG_ORG: kubeflow
+  IMG_REPO: model-registry-ui
```

Comment:
relies on mechanism from :

https://github.com/kubeflow/model-registry/blob/21b578e1256a7d2a157d89fcd17b950a65a2e3f8/Makefile#L40-L41

Reply:
thanks for highlighting this, I would have missed that easily 😁 

/lgtm

Reply:
glad to help, I did also forgot. maybe we could consider ways to refactor this too going fwd, separately from this PR, wdyt?

Reply:
I agree